bcda/models/SwaggerStructs.go

```diff
@@ -179,6 +179,7 @@ type GroupIDParam struct {
 	// ID of group export
 	// in: path
 	// required: true
+	// enum: all,runout
```

Comment:
What happens if this is requested in `prod` now?  Won't it fail, since there are zero runout files ingested for the ACOs as of today?

Reply:
It'll fail for all of the non-synthetic ACOs. I think that'll be OK though since we want to expose this functionality in sandbox